frame/support/procedural/src/lib.rs

```diff
@@ -479,6 +480,63 @@ pub fn pallet(attr: TokenStream, item: TokenStream) -> TokenStream {
 	pallet::pallet(attr, item)
 }
 
+/// An attribute macro that can be attached to a (non-empty) module declaration. Doing so will
+/// designate that module as a benchmarking module.
+///
+/// See `frame_support::benchmarking` for more info.
+#[proc_macro_attribute]
+pub fn benchmarks(attr: TokenStream, tokens: TokenStream) -> TokenStream {
+	benchmark::benchmarks(attr, tokens, false)
+}
+
+/// An attribute macro that can be attached to a (non-empty) module declaration. Doing so will
+/// designate that module as an instance benchmarking module.
+///
+/// See `frame_support::benchmarking` for more info.
```

Comment:
Same reason here. Those items aren't accessible because we're in support procedural, so best we can do is just mention their name like we do in the pallet macros